src/components/molecules/UserMenu/UserMenuComponent.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Menu, MenuItem } from "@mui/material";
+import { FC, MouseEvent } from "react";
+import { UserMenuButton } from "@/components/atoms";
+
+type Props = {
+  anchorEl: null | HTMLElement;
```

Comment:
It will require you to pass null to anchorEl even if you don't want to pass it
![image](https://github.com/tomaszstefaniuk/take-for-help/assets/50902912/8a2e1a46-cb35-4fd3-9035-49dfbdf28a77)

change it to 
```js
anchorEl?: HTMLElement;
```